### ISSUE TITLE
GeneratedByEngineVersion 2.0以下无法工作

### ISSUE THREAD

**Issue by mlican** [qingshan/mdict_reader#3]

**Comment by qingshan** [comment]
Yes, `mdict_reader` only works for `GeneratedByEngineVersion=2.0`. but if you could provide .mdx file with `GeneratedByEngineVersion < 2.0`, I could implement it.

**Comment by anhtuan23** [comment]
@qingshan Hello, here is a file with `GeneratedByEngineVersion == 1`. It would be great if you can support it.
[jmdict.mdx.zip](https://github.com/qingshan/mdict_reader/files/10536601/jmdict.mdx.zip)

**Comment by qingshan** [comment]
@anhtuan23 Now support mdict file with `GeneratedByEngineVersion < 2.0`. but currently lzo compression is not supported. Because your uploaded mdict file is using lzo compression, so still need to implement lzo decompression to support the file.
